Google.Api.Gax/Platform.cs

```diff
@@ -209,6 +209,11 @@ public class KubernetesData
             /// </summary>
             public string PodName { get; set; }
 
+            /// <summary>
+            /// The kubernetes namespace name
```

Comment:
Given that this documentation and that for PodName are public, could we capitalize the K in Kubernetes?